### PR TITLE
feat: add sonar badge [4]

### PR DESCRIPTION
**Pull request by adong**
## Context

Add sonar badge display to the pipeline page and configurable in pipeline option page

## Objective
image
image

Tests pending
## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jithine** [comment]
what we need is for a way to get the sonar badge information auto populated. I wonder if the coverage bookend can do this. It does make an API call explicitly to create the project, I think that API call should update the Pipeline information as well.

**Comment by adong** [comment]
> what we need is for a way to get the sonar badge information auto populated. I wonder if the coverage bookend can do this. It does make an API call explicitly to create the project, I think that API call should update the Pipeline information as well.

Okay, maybe in the coverage bookend script, we can set this url inside the `pipeline.badge.sonar`

**Review by tkyi — DISMISSED** [review]

File: app/components/pipeline-header/component.js [hunk 1]
```diff
@@ -26,19 +48,30 @@ export default Component.extend({
       };
     }
   }),
-  sameRepoPipeline: computed('pipeline', {
+  sameRepoPipeline: computed('pipeline.scmRepo.name', 'pipeline.{id,scmUri}', {
```
Comment (jithine) [inline app/components/pipeline-header/component.js:51]: what are these changes for ?

Comment (adong) [reply app/components/pipeline-header/component.js:51]: I think these codes are added when rebased with master

**Pull request merged**